scripts/time_plot.py

```diff
@@ -3358,7 +3358,14 @@ def send_websock_cmd(cmd, handlerkey):
     try:
         if handlerkey in websockrequest_username:#else skip - connection may have gone midway through a send update
             frame=u"/*exec_user_cmd*/ function callme(){%s; return;};callme();" % cmd;#ensures that vectors of data is not sent back to server!
-            handlerkey.write_message(frame)
+            if handlerkey.ws_connection is None or handlerkey.ws_connection.is_closing():
+                logger.warning("Stream closing for %s@%s", websockrequest_username[handlerkey], handlerkey.request.remote_ip)
+                deregister_websockrequest_handler(handlerkey)
+            else:
+                handlerkey.write_message(frame)
+    except tornado.iostream.StreamClosedError:
```

Comment:
You could also combine the two exception clauses if it is basically the same thing, like so:
```python
except (tornado.websocket.WebSocketClosedError, tornado.iostream.StreamClosedError):
    ...
```

Reply:
Ok will do